systemtest/src/main/java/io/strimzi/systemtest/matchers/LogHasNoUnexpectedErrors.java

```diff
@@ -79,7 +79,9 @@ enum LogIgnoreList {
         ASSEMBLY_OPERATOR_RECONCILIATION_TIMEOUT("ERROR .*AssemblyOperator:[0-9]+ - Reconciliation.*[fF]ailed.*"),
         WATCHER_CLOSED_EXCEPTION("ERROR AbstractOperator:.+ - Watcher closed with exception in namespace .*"),
         CONCURRENT_RESOURCE_DELETION("io.strimzi.operator.cluster.operator.resource.ConcurrentDeletionException"),
-        RECOVERY_STS_DELETION("java\\.lang\\.IllegalStateException: Can't wait for StatefulSet: recovery-cluster-(kafka|zookeeper) in namespace: recovery-cluster-test to scale. Resource is no longer available.");
+        RECOVERY_STS_DELETION("java\\.lang\\.IllegalStateException: Can't wait for StatefulSet: recovery-cluster-(kafka|zookeeper) in namespace: recovery-cluster-test to scale. Resource is no longer available."),
+        // error that pods already exist when doing reconciliation of StrimziPodSets
+        RECONCILIATION_PODSET_ALREADY_EXISTS("ERROR StrimziPodSetController:[0-9]+ - Reconciliation.*[fF]ailed.*already exists.*");
```

Comment:
This seems quite wide. When and how does it happen?

Reply:
It usually happen during the upgrade from `-UseStrimziPodSets` to `+UseStrimziPodSets` FG

Reply:
And how do the full error messages / logs look like?

Reply:
```
12:48:26 ERROR StrimziPodSetController:323 - Reconciliation #129(watch) StrimziPodSet(infra-namespace/my-cluster-kafka): StrimziPodSet my-cluster-kafka in namespace infra-namespace reconciliation failed\nio.fabric8.kubernetes.client.KubernetesClientException: Failure executing: POST at: https://172.30.0.1/api/v1/namespaces/infra-namespace/pods. Message: pods \"my-cluster-kafka-0\" already exists. Received status: Status(apiVersion=v1, code=409, details=StatusDetails(causes=[], group=null, kind=pods, name=my-cluster-kafka-0, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=pods \"my-cluster-kafka-0\" already exists, metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=AlreadyExists, status=Failure, additionalProperties={}).\n\tat io.fabric8.kubernetes.client.KubernetesClientException.copyAsCause(KubernetesClientException.java:238) ~[io.fabric8.kubernetes-client-api-6.2.0.jar:?]\n--\n\tat java.lang.Thread.run(Thread.java:829) ~[?:?]\nCaused by: io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: POST at: https://172.30.0.1/api/v1/namespaces/infra-namespace/pods. Message: pods \"my-cluster-kafka-0\" already exists. Received status: Status(apiVersion=v1, code=409, details=StatusDetails(causes=[], group=null, kind=pods, name=my-cluster-kafka-0, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=pods \"my-cluster-kafka-0\" already exists, metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=AlreadyExists, status=Failure, additionalProperties={}).\n"
```

Reply:
Maybe you should raise it as issue in the first place including the full log etc.

Reply:
Well we discussed this, but I will create an issue, so you can take a look on it.

Reply:
Thanks.

Reply:
Maybe you could also link the issue to the whitelist item through a comment.

Reply:
Yeah, sure thing